src/crypto-api/CryptoEventHandlerMap.ts

```diff
@@ -18,6 +18,7 @@ import { CryptoEvent } from "./CryptoEvent.ts";
 import { VerificationRequest } from "./verification.ts";
 import { UserVerificationStatus } from "./index.ts";
 import { RustBackupCryptoEventMap } from "../rust-crypto/backup.ts";
+import { DehydratedDevicesEventMap } from "../rust-crypto/DehydratedDeviceManager.ts";
```

Comment:
`crypto-api` really shouldn't be pulling things in from `rust-crypto`. `RustBackupCryptoEventMap` is setting a poor example here.

I think the right way to do this is to add the values from `DehydratedDevicesEventMap` to `CryptoEventHandlerMap`, and then redefine `DehydratedDevicesEventMap` as `Pick<CryptoEventHandlerMap, DehydratedDevicesEvents>` or something like that.